sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/administration/DocumentModelAdministrationAsyncClient.java

```diff
@@ -868,6 +880,327 @@ public PagedFlux<OperationSummary> listOperations() {
+     *             documentModelInfo.getCreatedOn&#40;&#41;&#41;&#41;;
+     * </pre>
+     * <!-- end com.azure.ai.formrecognizer.documentanalysis.administration.DocumentModelAdminAsyncClient.listDocumentClassifiers -->
+     *
+     * @return {@link PagedFlux} of {@link DocumentClassifierDetails document classifiers} on the Form Recognizer account.
+     */
+    @ServiceMethod(returns = ReturnType.COLLECTION)
+    public PagedFlux<DocumentClassifierDetails> listDocumentClassifiers() {
+        try {
+            return new PagedFlux<>(() -> withContext(this::listFirstPageClassifiers),
+                continuationToken -> withContext(context -> listNextPageClassifiers(continuationToken, context)));
+        } catch (RuntimeException ex) {
+            return new PagedFlux<>(() -> monoError(logger, ex));
+        }
+    }
+
+    private Mono<PagedResponse<DocumentClassifierDetails>> listFirstPageClassifiers(Context context) {
+        return documentClassifiersImpl.listClassifiersSinglePageAsync(context)
+            .doOnRequest(ignoredValue -> logger.info("Listing information for all models"))
+            .doOnSuccess(response -> logger.info("Listed all models"))
+            .doOnError(error -> logger.warning("Failed to list all models information", error))
+            .onErrorMap(Transforms::mapToHttpResponseExceptionIfExists)
+            .map(res -> new PagedResponseBase<>(
+                res.getRequest(),
+                res.getStatusCode(),
+                res.getHeaders(),
+                res.getValue().stream()
+                    .map(documentClassifierDetails -> Transforms.fromInnerDocumentClassifierDetails(documentClassifierDetails))
+                    .collect(Collectors.toList()),
+                res.getContinuationToken(),
+                null));
+    }
+
+    private Mono<PagedResponse<DocumentClassifierDetails>> listNextPageClassifiers(String nextPageLink, Context context) {
+        if (CoreUtils.isNullOrEmpty(nextPageLink)) {
+            return Mono.empty();
+        }
+        return documentClassifiersImpl.listClassifiersNextSinglePageAsync(nextPageLink, context)
+            .doOnSubscribe(ignoredValue -> logger.info("Retrieving the next listing page - Page {}", nextPageLink))
+            .doOnSuccess(response -> logger.info("Retrieved the next listing page - Page {}", nextPageLink))
+            .doOnError(error -> logger.warning("Failed to retrieve the next listing page - Page {}", nextPageLink,
+                error))
+            .onErrorMap(Transforms::mapToHttpResponseExceptionIfExists)
+            .map(res -> new PagedResponseBase<>(
+                res.getRequest(),
+                res.getStatusCode(),
+                res.getHeaders(),
+                res.getValue().stream()
+                    .map(documentClassifierDetails -> Transforms.fromInnerDocumentClassifierDetails(documentClassifierDetails))
+                    .collect(Collectors.toList()),
```

Comment:
NIT: It is a good practice to not use Collectors.toList(). It doesn't know the size of the initial list which may result in some resizing but it's not much of a win at all.